corehq/apps/reports/views.py

```diff
@@ -1756,7 +1756,14 @@ def _error(msg):
         instance = _get_location_safe_form(domain, request.couch_user, instance_id)
         context = _get_form_context(request, domain, instance)
         if not instance.app_id or not instance.build_id:
-            return _error(_('Could not detect the application/form for this submission.'))
+            deviceID = instance.metadata.deviceID
+            if deviceID and deviceID == 'Formplayer':
```

Comment:
@benrudolph I couldn't find where this was set. Is there a constant I can pull from for this?